docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,533 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4dd46d67",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "issue_to_visualize = issue_idx[10]  # change this to view other images\n",
+    "class_names = {\"0\": \"car\", \"1\": \"chair\", \"2\": \"cup\", \"3\":\"person\", \"4\": \"traffic light\"}\n",
+    "\n",
+    "label = labels[issue_to_visualize]\n",
+    "prediction = predictions[issue_to_visualize]\n",
+    "score = scores[issue_to_visualize]\n",
+    "image_path = IMAGE_PATH + label['seg_map']\n",
+    "\n",
+    "print(image_path, '| idx', issue_to_visualize , '| label quality score:', score, '| is issue: True')\n",
+    "visualize(image_path, label=label, prediction=prediction, class_names=class_names, overlay=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de0d7205",
+   "metadata": {},
+   "source": [
+    "The visualization depicts the given label (original image annotation which cleanlab identified as problematic) in red on the left and the model-predicted label in blue on the right. Each bounding box contains a class-index number in the top corner indicating which object class that bounding box was annotated/predicted to contain.\n",
+    "\n",
+    "This image has a **low** label quality score and is marked as an error. On closer inspection we notice the annotator missed a car in the back right corner that the model identified. Additionally, the bounding box for the foreground car the woman is in is cut short a bit on the right. \n",
+    "\n",
+    "Notice examples where the predictions and labels are more similar have higher quality scores than those that are missmatched, and are less likeley to be marked as issues and the number of boxes is agnostic to the score.\n",
+    "\n",
+    "Since our model is not perfect, it incorrectly identified the passenger dog as a person however it was still able to identify issues in the scene. Better trained models will lead to better label error detection but you don't need a near perfect model to identify label issues.\n",
+    "\n",
+    "\n",
+    "### Different kinds of label issues identified by ObjectLab\n",
+    "Now lets view the first few images in our vaidation dataset that are clearly marked as issues and see what various inconsistencies between the `given` and `predicted` label we can spot. \n",
+    "\n",
+    "The monitor image is missing annotations for a cup and a person working on a chair in the background. The birthday party image should have had  individual bounding boxes around each kid (the COCO guidelines state only groups with 10+ objects of the same type can be a \\\"crowd\\\" bounded by a single box). Some cups were overlooked by annotators of the dinner party image, and the empty chair was annotated incorrectly. Finally, the chairs were overlooked by annotators from the foreground of the abandoned school bus images."
```

Comment:
I think this paragraph should be deleted? Most of the observations have been repeated below where the images are shown (the monitor, corn-on-cob examples etc.). And I am not seeing an image of the abandoned school bus

Reply:
good catch! This is why it is dangerous to discuss specific images in the text and I recommend against it unless there are robust tests for it in the final hidden cell.  

FYI hold off on reviewing so closely though, there are still potentially some bigger pending changes incoming